src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -9,6 +11,14 @@ namespace GitHub.Services
     [PartCreationPolicy(CreationPolicy.Shared)]
     public class PullRequestService : IPullRequestService
     {
+        static readonly string[] TemplatePaths = new[]
+        {
+            "PULL_REQUEST_TEMPLATE.md",
```

Comment:
I believe that github.com supports `PULL_REQUEST_TEMPLATE.txt` as well. I don't think it checks the file extension.